proto/container/container.proto

```diff
@@ -19,18 +19,30 @@ option java_outer_classname = "ApplicationServiceProto";
 
 package container;
 
-// The greeting service definition.
 service ContainerService {
-  // Sends a greeting
   rpc CreateContainer (CreateContainerRequest) returns (CreateContainerReply) {}
+	rpc ImageStatus (ContainerID) returns (ContainerStatus){  }
 }
 
-// The request message containing the user's name.
+
+
+
+
+
 message CreateContainerRequest {
```

Comment:
you might want to add some identifying information. aka, when I want to check on an image, how do I ask for x image? name? maybe `customer` and `image name`? we don't have to solve this in this PR, just some thoughts.